packages/spatial/src/transform/components/TransformComponent.ts

```diff
@@ -227,10 +233,32 @@ export const TransformComponent = defineComponent({
     transform.matrixWorld.compose(vec3, quat, scale)
   },
 
+  forward: (entity: Entity) => {
+    const matrixElements = getComponent(entity, TransformComponent).matrix.elements
```

Comment:
we should have an optional vec3 passed in to these functions that the output is put onto, rather than a single hidden global

Reply:
changed